index.html

```diff
@@ -109,9 +109,47 @@ <h2 class="multi">Multi-Post Stories</h2>
             </div>
         </div>
     </div>
+       
+        <!--POP UP WINDOW-->
+
+    <div id="modulo">
+        <div class="content">
+            <div class="heading">
+                <h2>Keeping track of hundreds  of components website</h2>
+                <div class="close">
+                    <i class="fa fa-times" aria-hidden="true"></i>
+                </div>
+            </div>
+            <div class="pop">
+                <ul class="pop1">
+                    <li  class="pop2">
+                        <button type="button">HTML</button>
+                    </li>
+                    <li  class="pop2"> <button type="button">BootStrap</button></li>
+                    <li  class="pop2"> <button type="button">Ruby on rails</button></li>
+                </ul>
+            </div>
+            
+            <div class="img-p-button">
+                <img class="image" src="./Desktop/pop.png" alt="live pop">
+                <div class="p-button">
+                    <p class="description">Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem
+                        Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took
+                        a galley of type and scrambled it to make a type specimen book. It has survived not only five
+                        centuries, but also the leap into electronic typesetting, remaining essent</p>
+                    <div class="double-button">
+                        <a class="first-button" href="/">See Live <img src="./Desktop/rondom.png" alt="github source">
+                        </a>
+                        <a class="first-button" href="/">See Source <img class="popimg" src="./Desktop/popbtn.png" alt="github web"> </a>
+                    </div>
+                </div>
+            </div>
+        </div>
```

Comment:
- [ ] Following the project requirements, The pop-up should be created dynamically with the information from the projects array. It should be added via JavaScript.

Reply:
Thank you very much for  the CR